extensions/vertx/runtime/pom.xml

```diff
@@ -18,6 +18,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This feels wrong to me. Given that RESTEasy now depends on vert.x maybe it would make more sense to move the MessageBodyWriter implementations into the RESTEasy module?